config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
+  port: 5432
   user: postgres
-  password: postgres
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
```

Comment:
Yes, definitely - possibly by sourcing an `.env` file or similar. This is probably related to #838.